content/applications/sales/crm/track_leads/marketing_attribution.rst

```diff
@@ -0,0 +1,142 @@
+the search bar to see the list of filtering and grouping parameters.
+
+:guilabel:`Filters`, located in the left column of the search options, can be used to keep only
+the results that fit the filter. For example, selecting the :guilabel:`Won` filter will only
+show won leads in the attribution report. :guilabel:`Group By`, found in the middle column, is
+used to organize the results into groups and can be used with or without filters.
+
+.. image:: marketing_attribution/search-results-multiple-options.png
+   :align: center
+   :alt: Select any number of filters and groups in the search options.
+
+.. tip::
+    Setting multiple Group By options will create nested groups according to which is selected
+    first. For example, selecting :guilabel:`Medium` followed by :guilabel:`Source` and then
+    ::guilabel:`Campaign` in :guilabel:`Group By` will sort all results *first* by medium, *then* by
+    the specific sources in each medium, followed by the campaigns in each source.
+
+    This can be verified by looking at the direction of the carat in the group tile in
+    the search bar.
+
+    .. image:: marketing_attribution/group-by.png
+       :align: center
+       :alt: The text in the tile is `Country > City`, showing that city is a subgroup of country.
+
+.. example::
+    For a useful first report:
+
+    #. Select the :guilabel:`Active` filter to view only leads that are still marked as active.
+    #. Select :guilabel:`Source` followed by the :guilabel:`City` or :guilabel:`Country`
+       groups depending on which grouping is more relevant.
+
+    .. image:: marketing_attribution/campaign-and-country-groups.png
+       :align: center
+       :alt: Each lead is now sorted by source, followed by city or country.
+
+    This report will contain all active leads grouped first by the source of the lead, then the
+    country or city each lead is from. This is useful to see the density of active opportunities
+    sorted by location.
+
+    With this data, marketing campaigns such as conferences or billboards can be targeted to the
+    locations generating the largest amount of potential revenue. Similarly, more attention can be
+    put toward increasing outreach in locations where existing marketing campaigns are less
+    effective.
+
+Export reports
+==============
+
+To set measures of the report, begin by navigating to the :icon:`oi-view-pivot`
+:guilabel:`(pivot view)`. Click the :guilabel:`Measures` button to view the available measures of
+the report. Select the desired measures, and verify the measures, filters, and groups are all
+displayed correctly in the pivot table.
+
+Next, click on the :icon:`fa-cog` :guilabel:`(settings)` button to
+the right of :guilabel:`Lead Analysis` in the top-left of the page to view the available export
+options.
+
+To export the report in the Odoo *Documents* app, navigate to :menuselection:`Settings app -->
+Spreadsheet --> Insert list in spreadsheet`. A menu will appear with export options.
```

Comment:
Does the user *have to* leave the CRM app (and report they just built) to do this? Or, can they simply click the 'Insert in Spreadsheet' button? Also, in the previous paragraph, you instructed the reader to open the Actions (gear) menu...but then didn't break down their options on that menu, nor did you provide information about what they should select... I think it might benefit the overall doc if some added clarification and elaboration was placed in that portion...just to round things out, so to speak...

Reply:
I'm also not seeing a clickable header menu option, feature, or section on the Settings app with the name: Spreadsheet.

Reply:
Perhaps, instead of "Settings app..." you meant to say "Settings icon" (which I have changed to the 'Actions' (gear) above)
In that case, a potential rewording of this section could be...

From the resulting :guilabel:`Actions` :icon:`fa-cog` :guilabel:`(gear) drop-down menu, select :menuselection:`Spreadsheet --> Link menu in spreadsheet'.

Was that what you intended here? If not, just let me know. But, that's what I'm seeing on the runbot I am using for this review.

![Screenshot 2024-07-15 093623](https://github.com/user-attachments/assets/2304e4ec-b037-49ae-8429-b1bea43c86af)

Reply:
UPDATE: I think I see where the "disconnect" happened here...

So, at the top of this section, you instruct the reader to enter Pivot view, and make some customizations. 

Then, you tell them to open the Actions (gear) drop-down menu, but don't tell them why/what to select...

After that, you (I believe mistakenly) instruct them to go to the Settings app, but I don't think that's necessary, as the 'Insert in Spreadsheet' options are available directly from the reporting page.

Following that, you have different results and options via the Spreadsheet directions, and I think that's because the directions you have provided here, are for things in a List view (*not* Pivot view, which you have just told the reader to enter). That's why your instructions/options are different.

When in list view, the "Name of the list..." field and "Insert the first ______ records" field appears on the pop-up...that does not occur when in Pivot...

So, during your revisions, give this section another read-through/run-through, and make the necessary adjustments to ensure that everything lines and works as intended. Thanks! 👍